src/python/pants/backend/terraform/dependencies_test.py

```diff
@@ -24,7 +26,7 @@
 
 def _do_init_terraform(
     rule_runner: RuleRunner, standard_deployment: StandardDeployment, initialise_backend: bool
-) -> DigestContents:
+) -> Tuple[DigestContents, DigestEntries]:
```

Comment:
Since you included `from __future__ import annotations`, you can also use `tuple`, which is the pattern we use in most other placesm instead of `typing.Tuple`.

Reply:
I keep forgetting this and being surprised every time